src/pages/studyView/charts/ChartContainer.tsx

```diff
@@ -349,6 +369,14 @@ export class ChartContainer extends React.Component<IChartContainerProps, {}> {
         } as ChartControls;
     }
 
+    get xAxisOnTop(): boolean | undefined {
+        if (this.chartType == ChartTypeEnum.MUTATION_DIAGRAM) {
+            return this.props.store.getMutationStore(
+                this.props.chartMeta.uniqueKey
+            ).isXAxisOnTopSelected;
+        }
+    }
```

Comment:
not sure if we really need to support the case where x-axis is on top in study view. probably it's only useful in the comparison view.